web_src/less/_package.less

```diff
@@ -0,0 +1,3 @@
+.container-labels td:nth-child(2) {
+  word-break: break-word;
```

Comment:
```suggestion
  overflow-wrap: anywhere
```

`word-break: break-word` is deprecated [as per MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/word-break)

Reply:
`overflow-wrap: anywhere;` works, thanks!

Reply:
Ah yes, it needs `anywhere` because the string don't contain whitespace.